modules/capability/depinject.go

```diff
@@ -0,0 +1,51 @@
+package capability
+
+import (
+	"cosmossdk.io/core/appmodule"
+	"cosmossdk.io/depinject"
+	storetypes "cosmossdk.io/store/types"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+
+	modulev1 "github.com/cosmos/ibc-go/api/capability/module/v1"
+	"github.com/cosmos/ibc-go/modules/capability/keeper"
+)
+
+var _ depinject.OnePerModuleType = AppModule{}
+
+// IsOnePerModuleType implements the depinject.OnePerModuleType interface.
+func (AppModule) IsOnePerModuleType() {}
+
+func init() {
+	appmodule.Register(
+		&modulev1.Module{},
+		appmodule.Provide(ProvideModule),
+	)
+}
+
+type ModuleInputs struct {
+	depinject.In
+
+	Config *modulev1.Module
+	Cdc    codec.Codec
+	Key    *storetypes.KVStoreKey
+}
+
+type ModuleOutputs struct {
+	depinject.Out
+
+	CapabilityKeeper *keeper.Keeper
+	Module           appmodule.AppModule
+}
+
+// ProvideModule returns the capability module outputs for dependency injection
+func ProvideModule(in ModuleInputs) ModuleOutputs {
+	capabilityKeeper := keeper.NewKeeper(
+		in.Cdc,
+		in.Key,
+		in.Key, // TODO: where do we get mem key from?
```

Comment:
Let's add `MemKey *storetypes.MemoryStoreKey` to `ModuleInputs` I think! 

Example from v0.47 when x/capability was in the sdk: https://github.com/cosmos/cosmos-sdk/blob/v0.47.10/x/capability/module.go#L191

Reply:
https://github.com/cosmos/ibc-go/pull/6158/commits/a79e0534058a8e628431360043bdbdee9c0b6d75